### ISSUE TITLE
Error training after saving the `latest` model.

### ISSUE THREAD

**Issue by Naruto-Sasuke** [junyanz/pytorch-CycleGAN-and-pix2pix#253]
I think the latest commit cause the problem.
https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/commit/e2b72f5da0a45f9bc81fd7f926a1a6205bb3b182

A quick way to solve swap these two lines of code https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/models/base_model.py#L72:
```python
net.cuda(self.gpu_ids[0])
torch.save(net.module.cpu().state_dict(), save_path)
```
However, this does not sovle multigpu support error.

**Comment by taesung89** [comment]
We fixed the issue (ea3b646) following your suggestion. We will investigate the multigpu case as well.

**Comment by akshitamittel** [comment]
Yes, that's what I ended up doing. Thank you!
